File: src/main/java/io/saso/dash/services/ServiceManager.java
package io.saso.dash.services;

import io.netty.channel.ChannelHandlerContext;
import io.saso.dash.database.entities.LiveToken;

public interface ServiceManager
{
    void start(ChannelHandlerContext ctx, LiveToken liveToken) throws Exception;

    void stop(ChannelHandlerContext ctx, LiveToken liveToken) throws Exception;
}

File: src/main/kotlin/io/saso/dash/services/ServiceManager.kt
package io.saso.dash.services

import io.netty.channel.ChannelHandlerContext
import io.saso.dash.database.entities.DBLiveToken

public interface ServiceManager
{
    fun start(ctx: ChannelHandlerContext, liveToken: DBLiveToken)

    fun stop(ctx: ChannelHandlerContext, liveToken: DBLiveToken)
}
